invenio_oauthclient/templates/invenio_oauthclient/settings/index.html

```diff
@@ -13,6 +13,7 @@
 
 {% set panel_title = _("Linked accounts") %}
 {% set panel_icon = "fa fa-link" %}
+{% set num_linked_services = services|selectattr("account")|list|length %}
```

Comment:
I am not sure we need the check for local login. Isn't it enough to check only if there is one or multiple external accounts?
Minor suggestion:
```suggestion
{% set num_linked_services = services|selectattr("account")|list|length %}
{% set can_disconnect = num_linked_services > 1 or not local_login_impossible %}

....

{%- if s.account and can_disconnect -%}
```

Reply:
do you mean we should always disable the disconnect of the last external auth, regardless of whether or not a local login is possible?

Reply:
If there is only 1 external auth, then maybe the button should be disabled independently of the local login. Or am I missing something?

Reply:
sure, we can ignore the possibility for local login and just prohibit disconnecting the last remaining external auth anyway
that would probably render some of the changes in this PR irrelevant, and the code could be slightly simplified again
i'll ask again in the auth channel in discord, to make sure we're all on the same page